guides/getting_started_hassio.rst

```diff
@@ -2,36 +2,25 @@ Getting Started with ESPHome and Home Assistant
 ===============================================
 
 .. seo::
-    :description: Getting Started guide for installing ESPHome as a Home Assistant add-on and creating a basic configuration.
+    :description: Getting Started guide for installing ESPHome Dashboard as a Home Assistant add-on and creating a basic configuration.
     :image: home-assistant.png
 
 ESPHome is the perfect solution for creating custom firmware for
 your ESP8266/ESP32 boards. In this guide we’ll go through how to setup a
-basic "node" by use of the Home Assistant add-on.
+basic "node" using the ESPHome Dashboard, installed as a Home Assistant add-on.
 
 But first, here's a very quick introduction to how ESPHome works:
-ESPHome is a *tool* which reads in a YAML configuration file (just like Home Assistant)
-and creates a custom firmware binary. The tool also has many helpers that simplify flashing devices (uploading the new binary file)
-and aim to make managing your ESP boards as simple as possible. Once you have added devices
-or sensors in ESPHome's configuration, they will even automatically show up in Home
-Assistant's UI.
+ESPHome is a *tool* which aims to make managing your ESP boards as simple as possible. It reads in a YAML configuration file (just like Home Assistant) and creates custom firmware which it installs on your ESP device. Devices or sensors added in ESPHome's configuration will automatically show up in Home Assistant's UI.
 
-Installation
-------------
-
-Installing the ESPHome Home Assistant add-on is easy. Navigate to the Supervisor
-panel in your Home Assistant frontend, then enter ``ESPHome`` in the searchbar of the "Add-on Store"
-tab.
-
-.. figure:: images/hassio_repo.png
+Installing ESPHome Dashboard
+----------------------------
 
-Click on ESPHome, then INSTALL.
+The ESPHome Dashboard can be installed as a Home Assistant add-on, which you can find in the add-on store in the Supervisor panel. Open it using the following button then click on INSTALL:
 
-.. figure:: images/hassio_addons_section.png
+.. image:: https://my.home-assistant.io/badges/supervisor_addon.svg
+   :target: https://my.home-assistant.io/redirect/supervisor_addon/?addon=a0d7b954_esphome
```

Comment:
Can we open this in a new tab? This currenty will take the user away from the guide...